### PR TITLE
Add options

### PR DESCRIPTION
**Pull request by initFabian**
Please review @yfujiki @sallyransom.

Steps:
- Read README file first so you have a understanding and leave comments if something is still unclear. This README needs to be impeccable. 
- Review these code changes and also leave comments if variable or class names should change.

### CODE REVIEW

File: Cely Demo/AppDelegate.swift [hunk 1]
```diff
@@ -16,7 +16,16 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
     var window: UIWindow?
 
     func application(_ application: UIApplication, didFinishLaunchingWithOptions launchOptions: [UIApplicationLaunchOptionsKey: Any]?) -> Bool {
-        Cely.setup(with: window, forModel: User.ref, requiredProperties:[.Token])
+
+        Cely.setup(with: window!, forModel: User.ref, requiredProperties: [.Token], withOptions: [
+            .LoginCompletionBlock: { (username: String, password: String) in
+                if username == "asdf" && password == "asdf" {
```
Comment (yfujiki) [inline Cely Demo/AppDelegate.swift]: Why do you do this check here? Login is completed already, so the check has been already done before this block is called, no?

File: Cely/CelyLoginViewController.swift [hunk 2]
```diff
@@ -32,9 +32,9 @@ class CelyLoginViewController: UIViewController {
         if let fields = textFields,
             fields.flatMap({$0}).reduce(true, {$0 && !$1.text!.isEmpty}) {
 
-            Cely.save(usernameField?.text, forKey: "username")
-            Cely.save("token 29585843899", forKey: "token", securely: true)
-            Cely.changeStatus(to: .LoggedIn)
+            if let username = usernameField?.text, let password = passwordField?.text {
+                Cely.loginCompletionBlock?(username, password)
```
Comment (yfujiki) [inline Cely/CelyLoginViewController.swift]: Hm, okay. Cell does know username and password at this point...

Can you just save username and password, and change login status at this point? Then user doesn't need to do them in the completion block.

Comment (initFabian) [inline Cely/CelyLoginViewController.swift]: Oh btw, I just realized this was inside of the `CelyDemo`(Which is the example app) target. so all this code doesnt actually go into the framework.

Comment (yfujiki) [inline Cely/CelyLoginViewController.swift]: Oh `CelyLoginViewController` is part of the demo? I am confused. Which one is the login screen that Cely provides, then?

Comment (initFabian) [inline Cely/CelyLoginViewController.swift]: @yfujiki sorry, the `AppDelegate.swift` is part of Cely Demo, my bad.

**Pull request merged**